### DISCUSSION TITLE
is Superslicer dead? (answer: no)

### DISCUSSION THREAD

**Discussion by nick9one1** [supermerill/SuperSlicer#3902] [General]
there hasnt been an update in a year...

**Comment by jeremytodd1** [comment]
Superslicer is so far behind other slicers at this point. I switched to OrcaSlicer a few weeks back and have absolutely no regrets. It's great being on a slicer that actually gets updates again.

**Comment by Bumblebiber** [comment]
It's pretty sad that @supermerill doesn't make any statement on this topic. I'm pretty sure he reads threads like this, being not the only one. 

I also checked out Orca, seems like a pretty good alternative.

**Comment by jhstatewide** [comment]
> there hasnt been an update in a year...

yes

**Comment by CuredPrusa** [comment]
First of all, Supermerill was providing us with a superb slicing software for years, ages in front of Cura, even Prusa. Excellent tool for sophisticated printing, with abundance of advanced options for fine details in printing behavior.  And all that for free! What happened, why updates stopped, I don't know, but that doesn't at all diminish his outstanding work and contribution to SERIOUS printing community. I have my projects running on SuperSlicer, which Cura  (even Prusa) can not touch. 

Now there is Orca... 
-fancy colors, 
-stripped down functionality of SS (still better than Prusa), 
-some added functions for greater comfort, 
-faster and more reliable in processing. 

Did Bambulab pay to Supermerill to withdraw and let them continue? I don't know. All I know is that my SuperSlicer_2.5.59.0 is gold now and I will cherish it until something close to its performance comes in sight.

**Reply by jeremytodd1** [reply]
Out of curiosity, what is Orca missing from SS that you use?

**Reply by CuredPrusa** [reply]
Well, I cant say in details because I didn't research much, once I have found that essentials for my projects are missing:

- there's no speed in percentage  
- around half of modifiers are missing
- automatic seam position is not as strict in positioning, not to mention the fine tuning of custom positioning (manual paint is fine though)

enough for me.

**Reply by mastercartman** [reply]
I often switch my PC to laptop and back. I permanently have issues with snyching in Orca-Slicer/BambuStudio. It's very often missing some printer or filament settings and I recongnize when the print started and looks "bad". My way with SuperSlicer is synching the config folder with G-Drive, which in BambuSlicer is not working either, because it strictly binds printprofiles to printers and to accounts, it often doesn't recognize new profiles from my last session on the other PC (I tweak Slicer settings A LOT).
So SuperSlicer was my No1, now its PrusaSlicer but I miss the overwhelming flood of possibilities SuperMerill brought to us... :(

**Reply by PhilBaz** [reply]
Why do people say this about Cura. Cura is extremely advanced with very fine control over slicing behaviour.  I use cura for years and have not found another slicer that can replace it. Sure, every software has some feature missing, or that we would like from another. But, people should stop repeating this falsehood. No slicer is currently far above Cura. Cura is extremely popular with new and experienced users for a reason.

**Reply by j-snijder** [reply]
Problem though is that v 5.4.0 has a major bug that prevents slicing when tree support is choosen and in some other configurations.
And yes, I always used Cura to, for many years, and I'm sad to see that Cura blew his popularity for lots of us.
Why and how is still a riddle to me, but when these kind of bugs keep coming up you have no choise as to go look for a replacement.

**Reply by CuredPrusa** [reply]
"Cura is popular" and that's all that there is to it. Convenient for sculptures and artifacts. But when it comes to a technical part which has to be engineered inside-out thoroughly, Cura just can not match the tools (options) which SS provides abundantly.

**Reply by PhilBaz** [reply]
Can you give a few solid examples?

**Reply by CuredPrusa** [reply]
I could but it would take to much of my time to type about it. Sorry. But you can explore some major attributes yourself, like modifiers in height and/or shape. However, if you're happy with Cura, stay with it. I started with Cura four years ago, switched to SS after a year, found a whole new world of possibilities in slicing and left all the frustrations behind. I check new versions now and then, just to make sure I don't miss something significant.

**Reply by jhstatewide** [reply]
@CuredPrusa How about just your favorite feature then?

**Reply by PhilBaz** [reply]
One, best example?

**Reply by CuredPrusa** [reply]
Abundant modifying options in height or shape, plus custom codes per layer (coming from Prusa).

**Reply by CuredPrusa** [reply]
![Screenshot 2023-08-24 23 45 02](https://github.com/supermerill/SuperSlicer/assets/98842248/dfc6cdb8-7002-43f4-83f2-e6bd4f877d83)

**Reply by CuredPrusa** [reply]
I hope I can go to bed now. :)

**Reply by PhilBaz** [reply]
But surely 95% of this can be done in Cura with overlap modifiers.

**Reply by PhilBaz** [reply]
For me these settings are extremely important....
![Screenshot 2023-08-24 155257](https://github.com/supermerill/SuperSlicer/assets/21558241/71a27b85-309b-4fd1-976a-c1c8e1b1334f)

**Reply by CuredPrusa** [reply]
Maybe, if something changed meanwhile, but I doubt......


and dont call me surely.

**Reply by CuredPrusa** [reply]
@PhilBaz This doesn't have to be a competition. If you like your tool, use it the best you can. If you don't like it, find a better one. 
Here's the list of all settings in SS that you find "extremely important":
![Screenshot 2023-08-25 00 09 47](https://github.com/supermerill/SuperSlicer/assets/98842248/4e4441d9-6589-48aa-9058-8005bdea0a0f)
![Screenshot 2023-08-25 00 12 13](https://github.com/supermerill/SuperSlicer/assets/98842248/75789e41-ff43-48d8-8925-1e131e61f7b9)

**Reply by PhilBaz** [reply]
Yes, yes...it's not a competition, but a comparison. And if i search for every setting with the word brige I can make a larger looking list than I posted above. Maybe as long as your search. 

But the fact remains...to say SS is far more advanced than Cura simply doesn't seem to be true. I hear people say it but, have never seen a good example. Even here they appear to largely have similar functionality. Maybe one can do something the other doesn't and vice versa. But 95% the same.

**Comment by j-snijder** [comment]
Hi,I found next ti superslicer there is also pleccer, or superpleccer, which comes very close to superslicer.Op 11 aug. 2023 08:48 schreef CuredPrusa ***@***.***>:
First of all, Supermerill was providing us with a superb slicing software for years, ages in front of Cura, even Prusa. Excellent tool for sophisticated printing, with abundance of advanced options for fine details in printing behavior.  And all that for free! What happened, why updates stopped, I don't know, but that doesn't at all diminish his outstanding work and contribution to SERIOUS printing community. I have my projects running on SuperSlicer, which Cura  (even Prusa) can not touch.
Now there is Orca...
-fancy colors,
-stripped down functionality of SS (still better than Prusa),
-some added functions for greater comfort,
-faster and more reliable in processing.
Did Bambulab pay to Supermerill to withdraw and let them continue? I don't know. All I know is that my SuperSlicer_2.5.59.0 is gold now and I will cherish it until something close to its performance comes in sight.

**Reply by CuredPrusa** [reply]
That is very interesting, thank you!

**Comment by supermerill** [comment]
hi
I just resign my job.
I'm planning to go full time on slicer dev from november.
I found some companies that can hire me to do dev on superslicer.

**Reply by jhstatewide** [reply]
Merill, you make probably the best slicer everywhere. Thanks for all your work and thanks for keeping it open. You have some big fans and I hope you remember that for every annoying complaint you need to address there are lots of fans of SuperSlicer who can't wait to see what you do next.

Thanks a million!

**Reply by donbarthel** [reply]
That is great news. Can you tell us which companies have decided to fund SuperSlicer development? I'd like show them my appreciation.

**Reply by donbarthel** [reply]
I just sent a $20 "one-time sponsorship" through Github. I'll do it again if you follow through and again start releasing significant updates. Your announcement has made me happy.

**Reply by Eragon3650** [reply]
Long live the king

**Reply by timschneider** [reply]
Hi @supermerill, that sounds great! I'm looking forward to new great releases. If you don't mind me asking, what kind of job are you looking for? Are you looking for a permanent position or more of a freelance job? I can boost the sponsorship through Meltingplot if that helps, and if it's possible, I'd love to hire you feature-based. Some clients have special requirements and need special features. So I imagine we pay you to develop a particular feature, which is then available as an open source part of superslicer. Is that possible?

**Reply by MediumRare-notnerb** [reply]
I am so excited to see your work and development! Wish you the best

**Reply by theodiem** [reply]
Really awesome news. Congrats and looking forward to it !!!

**Reply by alchemy3d** [reply]
I really appreciate your work and hope this is the right way forward for you!
Take care!

**Reply by AuntJemima** [reply]
Are we in Lord of the Rings? Because it's The Return of the King!

**Reply by koisdigala** [reply]
Congrats, we're so happy to have you!

**Reply by j-snijder** [reply]
Happy face 😁 hopefully I can now definitif leave that Prusa Slicer behind me forever because its in no way near SuprSlicer quality

**Reply by NameOfTheDragon** [reply]
Awesome. I was worried there for a  while. SS must have been a tremendous drain if you were trying to work a  full-time job at the same time. I really hope this move pays off for you. You are making a difference.

**Reply by cpfarhood** [reply]
Best news I've heard all week.  Looking forward to seeing new releases.  Let me know if you need any test builds done on apple silicon, happy to help.

**Reply by macchinette** [reply]
The best news of my 2023!!!

**Reply by SABLISTER** [reply]
amazing!

**Reply by cryoglobe** [reply]
An even paid & Full time job to do Super Slicer development ( what are this / those companies winning in that ? ) 

But yes , that would be a great news ...  I had to go back to Prusa 2.6.1  ( hope you'll start the dev from that one, as there is great features as cut & plug , text , organic supports , ..

**Reply by trxtk89** [reply]
Manufacturers that intend on forking SS for their own printers and want a job well done
Foundations that have a lot of money to spare and want to push innovation

There's plenty of money to get out there, especially if you know how (& whom to) pitch your product.

**Reply by cryoglobe** [reply]
Perfect ... then ;)
for all of us

**Reply by j-snijder** [reply]
I do hope you keep it open source, lots of sharks outthere that would take us for their benefits and close the github repo.

Would be a bad thing 🥴

**Reply by VanessaE** [reply]
@j-snijder fortunately, SuperSlicer is GPL-licensed (well, AGPL), so no one can close it off, no matter how badly they want to.  There are over 1700 forks at the moment, so that ship has long since sailed and taken the rest of the fleet along with it. :wink:

**Reply by Arthur-de-Partuur** [reply]
@supermerill Super happy to hear this! I recently migrated to Orcaslicer and tried my best, but I stepped back to SuSi after a few tries...
Recently I discovered how to make a bundle.ini ( in de Vendor folder) and this gives me all the management of different filaments en prints/printers I need.... looking forward to yout next steps on the slicer!
As I am no coder / developer but still want to contribute where I can (donated some money a few weeks back), please let me know where I could contribute (other then money and filing bugs...)

**Reply by cristianku** [reply]
thank you for your work

**Reply by 5jvm0u4** [reply]
Hay, that company might be Prusa Research itself!

**Reply by themacboy** [reply]
Hello!

  There's a question that's on my mind... Why not join the OrcaSlicer project directly and merge the two?

**Reply by EvilGremlin** [reply]
Oh please no, babmu/Orca strip a lot of important settings. And UI is questionable.

**Reply by cryoglobe** [reply]
EvilGremlin ... Bambu choices are not Orca choices ... they already put back some features that were removed by Bambu.   So it could be a good jump ( idea to investigate ) to merge or retrofit some good points from Orca if Super Slicer stays in // track

**Reply by NiiXXiiN** [reply]
I might be late to find out the news but boy am i excited for it!! I am a big fan!
Superslicer is THE best slicer out there, the prints i get out from your slicer have no match with the others! I will be sure to support you in any way i can afford to.
Thank you for your work! I look forward to seeing what you come up with next or what features you integrate.

**Reply by ludovicurbain** [reply]
Sure why not.

If you're going full-time, would you consider starting from scratch?

So, I added a bunch of flow modifiers to superslicer to deal with recurring issues of it not being able to produce impeccable perimeters AND infill at the same extrusion multiplier on both Voron and RatRig printers.

Do you plan to keep superslic3r alive and should I try and make a pull request so this can be used by everyone, or should I just do a fork or whatever?

Side question: how much work is it trying to merge prusa slicer back into superslicer again and again whenever they release something useful?

![image](https://github.com/supermerill/SuperSlicer/assets/51236286/437b542c-7f5d-43ef-a9c5-1a8db1618c03)

**Reply by stein123456** [reply]
nice then it is hope maybe we can get profiles for the creality k1max, i have tried to play whit it but i feel that i wil miss using superslicer.
thanks dud for a very good slicer

**Reply by j-snijder** [reply]
Hope you re-consider going back to slic3r as a base for all and not go back to the poor quality of prusa. As prusa is way behind in print quality compared to the latest superslicer.

**Reply by xtianpaiva** [reply]
Is this November  comeback still a thing? Hope it is...

**Reply by siamak06** [reply]
I am very new to this particular Thread, but I have been following the Non-Planar Tool-Path Generation for a while,
Zip-o-mat 's Slic3r is an excellent job, nevertheless it occupies only on the last few layers, very good.

This in more on the Aesthetic part of the printing,

The Main reason of Non-Planar Slicing and printing is to increase the mechanical and Structural properties of the printed object, Isotropic is the main objective of Non-Planar Slicing and printing, as far as I can say and tried, Curvislicer, can more or less achieve this property, but it is Command line and indeed needs Post processing the G-code after having done the Slicing.
I think the Curviclicer as an Open-Source Slicer, and those who have the knowledge of Programming and Tool-Path generation, can do the integration into popular slicers, i.e SuperSlicer which is the most powerful one I can see and have used for some times now.

https://github.com/mfx-inria/curvislicer

Now that Supermerill had resigned and probably has time, would be good idea to try to do the integration of His Baby (SuperSlicer) and Curvislicer, It would be silly to suggest but would it be correct to say that keeping the slicing as successive 3D curves, with very low angle and very low amplitude that can randomly increase at the beginning and decrease in the end, this can easily solve the collision problem?
I hope one day one person, hoping to be SuperMerill, that will come with a brilliant solution of integrating the SuperSlicer and Curvislicer together for us the Mortal users.
Cheers and good luck.
Siamak

**Comment by KennyBeaudoin** [comment]
wow thats some sweet news
cant wait for the competition :)

**Comment by GeorgeGedox** [comment]
@supermerill I think it would be a good idea to update the readme to let everyone know about the current state and future plans as I've seen already plenty of people move back to PS for organic supports mostly or even Cura because of the recent updates. I'm sure most of them would like to know that SS is not abandoned and development will resume soon.

**Reply by cryoglobe** [reply]
yes .. make it public and known

**Comment by NameOfTheDragon** [comment]
Radical, I know, but did you know you can sponsor developers on GitHub?
![image](https://github.com/supermerill/SuperSlicer/assets/1268490/263e0440-1305-40aa-9828-61a354f780d0)
How about it?

**Comment by freighttt** [comment]
the return of merrill with proper compensation for his work on superslicer is the best 3d printing news of 2023!!!

**Comment by EvilGremlin** [comment]
@supermerill Welcome back! Cant wait for tree supports and new positioning from PS ported!

**Reply by mlewis-everley** [reply]
Some bug fixes would also be very much appreciated :-)

**Reply by cryoglobe** [reply]
yes :)  as the purge tower floating in the air :)

**Comment by macdarren** [comment]
I've been away from FDM printing for awhile and was sad to see that SuperSlicer seemed to have stalled out.  I am really happy to see that is not the case and maybe updates will resume shortly.  Two things I would like to see are better support structures and I think Supersclicer needs some minor GUI updates....  What I would like to see is a bit more flexibility in the main page quick options...like the support on and off or Brim on and off I'd like to be able to add a few others and would be great if the choices were user configurable since not everyone probably wants the same thing.  Next I would love to see a better way to show all the many options grouped differently....similar to above searches....Say I have problems with bridging or wall strength, I would like to click a button or two and see all the things that affect this and good explanations of what they do and how they affect the print and how they interact.  The pop ups are nice but need more info and are sometimes missing or incomplete or fail to pop...I would rather see just the items I need to see for a specific issue and clear (manual/documentation style) descriptions and ramifications right there under the option.  Thanks for all the hard work that goes into this project!

**Comment by hugoToscher** [comment]
Hi, I need to contact supermerill with some questions on the possibilities of non-planar slicing in Super Slicer. Is there any official contact information of his? Ideally, an email of some sort. @supermerill

**Comment by tsmith35** [comment]
> there hasnt been an update in a year...

Whoa... I was about to make the same post. 14 months is an eternity anymore. Merill should be able to grab code from Bambu Studio, OrcaSlicer, PrusaSlicer and the rest to catch up.

**Comment by supermerill** [comment]
hi
I left my job, now i'm full time on the slicer.

**Reply by hugoToscher** [reply]
Hi man, what's the best way to contact you? I need to talk to you about the potential use of super slicer for non-planar printing.

**Reply by Arthur-de-Partuur** [reply]
Cool,... I was already checking recently if the code had any recent changes (which it had)... Welcome back!

Any idea when you can share a bit of how the future plans look?

**Reply by katodo** [reply]
I managed to resist changing slicers, now I expect great things :) :) :)

**Reply by dfosantos** [reply]
Welcome back! I wish you good luck on this next stage of your life!
I'm really happy that you're back

**Comment by Vanilla009** [comment]
First new release is here!

**Reply by cryoglobe** [reply]
where ?

**Reply by tsmith35** [reply]
Release, or release candidate? v2.5.59.3?

**Reply by cryoglobe** [reply]
I suppose it's that one yes ... as it's from "septembre 2023"

I hope there will be a retrofit of the 2.6 - 2.7 new features

**Comment by Shefoofy** [comment]
> there hasnt been an update in a year...
No it's not dead sheesh.  Do you think the people posting here are all dead? no, just downloaded an update 07/2024

**Reply by tsmith35** [reply]
> > there hasnt been an update in a year...

> No it's not dead sheesh.  Do you think the people posting here are all dead? no, just downloaded an update 07/2024

The message you replied to was from July 2023, a year ago. Supermerill jumped back in around September 2023.

**Comment by sb53systems** [comment]
SuperSlicer is still a king, TY.
